src/quiz.js

```diff
@@ -2,67 +2,69 @@ var quiz = document.querySelector('#quiz');
 var questions = quiz.querySelector('#questions');
 var result = quiz.querySelector('#result');
 
-function Question() {
+class Question {
 
-     //The code below contains helper functions -- you don't need to edit this code
-    var questionContainer = copyTemplate('#question-template', questions);
-    var answersContainer = questionContainer.querySelector('.answers');
-    var imageContainer = questionContainer.querySelector('.imageForQuestion')
+    //The code below contains helper functions -- you don't need to edit this code
+    constructor() {
+        this.questionContainer = this.copyTemplate('#question-template', questions);
+        this.answersContainer = this.questionContainer.querySelector('.answers');
+        this.imageContainer = this.questionContainer.querySelector('.imageForQuestion')
+    }
 
-    function  getQuestionContainer() {
-        return questionContainer.querySelector('.question');
+    getQuestionContainer () {
+        return this.questionContainer.querySelector('.question');
     }
 
-    function createAnswerContainer() {
-        return copyTemplate('#answer-template', answersContainer);
+    createAnswerContainer() {
+        return this.copyTemplate('#answer-template', this.answersContainer);
     }
 
-    function createImageContainer() {
-        return copyTemplate('#image-template', imageContainer);
+    createImageContainer() {
+        return this.copyTemplate('#image-template', this.imageContainer);
     }
 
-    function setText(element, textToSet) {
+    setText(element, textToSet) {
         element.querySelector('.text').textContent = textToSet;
     }
 
-    function setRadioButton(element, name, value) {
+    setRadioButton(element, name, value) {
         var radioButtonElement = element.querySelector('.choice');
         radioButtonElement.setAttribute('name', name);
         radioButtonElement.setAttribute('value', value);
     }
 
-    function setImageSource(element, source) {
+    setImageSource(element, source) {
         var imageElement = element.querySelector('.image');
         imageElement.setAttribute('src', source)
     }
 
 
-    function copyTemplate(id, to) {
+    copyTemplate(id, to) {
         var element = document.querySelector(id).cloneNode(true);
         element.id = '';
         to.appendChild(element);
         return element;
     }
 
-    this.getSelectedAnswer = function () {
+    getSelectedAnswer() {
         return questionContainer.querySelector(':checked').value;
     }
 
-    this.setQuestion = function (questionText) {
-        var questionElement = getQuestionContainer();
-        setText(questionElement, questionText);
-    };
+    setQuestion(questionText) {
```

Comment:
I'd make all these arrow functions just to ensure `this` means what people think it does.

Reply:
Discussed offline. Not possible within classes :(